sklearn/ensemble/gradient_boosting.py

```diff
@@ -1344,6 +1355,8 @@ def predict_proba(self, X):
         score = self.decision_function(X)
         try:
             return self.loss_._score_to_proba(score)
+        except NotFittedError as e:
+            raise e
```

Comment:
Oh I didn't know  you could do that... Your comments teach me a lot... thanks! ;)